Data/Functor/Foldable.hs

```diff
@@ -59,27 +59,30 @@ module Data.Functor.Foldable
   , histo
   , ghisto
   , futu
+  , gfutu
   , chrono
   , gchrono
-  -- ** Distributive laws
-  , distCata
-  , distPara
-  , distParaT
-  , distZygo
-  , distZygoT
-  , distHisto
-  , distGHisto
-  , distFutu
-  , distGFutu
+  -- ** Gathering functions
+  , Gather
+  , gatherCata
+  , gatherPara
+  , gatherParaT
+  , gatherZygo
+  , gatherZygoT
+  , gatherHisto
+  , gatherGHisto
   -- * Unfolding
   , Corecursive(..)
   -- ** Combinators
   , gana
-  -- ** Distributive laws
-  , distAna
-  , distApo
-  , distGApo
-  , distGApoT
+  -- ** Scattering functions
+  , Scatter
+  , scatterAna
+  , scatterApo
+  , scatterGApo
+  , scatterGApoT
+  , scatterFutu
+  , scatterGFutu
```

Comment:
I took the opportunity to move the `futu`- and `gfutu`-related entries from the folding section to the unfolding section.